app_image_generator/plugins/slack.py

```diff
@@ -1,36 +1,36 @@
 import os
-from slacker import Slacker
 from app_image_generator.plugins.base import BasePlugin
 
 
 class SlackPlugin(BasePlugin):
     def __init__(self):
-        self.slacker = Slacker(os.environ.get("SLACK_TOKEN"))
-        self.channel = os.environ.get("SLACK_CHANNEL")
+        self.webhook_url = os.environ.get("SLACK_WEBHOOK_URL")
 
     def build_succeeded(self, app, output):
-        self.slacker.chat.post_message(self.channel or '#dev', 'New %s images available:' % app, username='app-image-generator', attachments=[
-            {
+        requests.post(self.webhook_url, data({
+            "attachments": [{
```

Comment:
no channel info, did you test this? :D

Reply:
When using webhooks, you define the channel in the webhook configuration.